doc/sphinx/users-guide/increasing-your-hitrate.rst

```diff
@@ -649,8 +674,12 @@ headers, then remove the headers from the client request in
 ``vcl_pass``, as shown above for pass.
 
 The hit-for-pass state ends when the "hit-for-pass TTL" given in the
-``return`` statement elapses. The next request for that object is then
-handled as an ordinary miss.
+``return`` statement elapses. As with passes, the response to a
+hit-for-pass fetch is never cached, even if it would otherwise fulfill
+conditions for cacheability.  So unlike hit-for-miss, it is not
+possible to end the hit-for-pass state ahead of time with a cacheable
```

Comment:
> So unlike hit-for-miss, it is not possible to end the hit-for-pass state ahead of time with a cacheable response.

I'm not sure it's impossible to escape hit-for-pass, what about `hash_always_miss`?